src/Main.java

```diff
@@ -1506,8 +1506,12 @@ else if (location.equals("N4G")) {
                     && canSuperSwim(inventory);
         }
         else if (location.equals("N4B")) {
+			/*
+			* Added MINOR GLITCHES logic to this chest.
+			* Perform a well-timed highjump waterclip to reach the room.
+			*/
             return canAccess("N4", inventory)
-                    && inventory.contains(Items.PUMP)
+                    && ((difficulty >= Difficulty.S_HARD && canSwim(inventory) && inventory.contains(Items.JUMP_BOOTS)) || inventory.contains(Items.PUMP))
```

Comment:
Sure, I can send you a sample pixel alignment that works for the charge while attempting to make this clip. That said, it's worth noting that waterclips are actually quite a bit more forgiving than normal walljumps. There tends to be multiple pixel sets that work for them (many more than walljumps and reverse walljumps). This one is only a little more finicky because of the fact that you need to do a high jump to succeed the clip.